README.md

```diff
@@ -617,6 +617,7 @@ API | Description | Auth | HTTPS | CORS |
 API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
 | [Countly](https://api.count.ly/reference) | Countly web analytics | No | No | Unknown |
+| [Creative Commons Catalog](https://api.creativecommons.engineering/) | Search among openly licensed and public domain works | No | Yes | Yes |
```

Comment:
```suggestion
| [Creative Commons Catalog](https://api.creativecommons.engineering/) | Search among openly licensed and public domain works | `OAuth` | Yes | Yes |
```

Reply:
I think most of the endpoints are only available through OAuth

Reply:
There is the option to use the API without an API key too, it is not strictly required :) but it's certainly more limited than the authenticated via.

Reply:
I see, I misread the introduction. Thanks for the explanations!